3rdparty/libsamplerate/src_sinc.c

```diff
@@ -1,18 +1,22 @@
 /*
-** Copyright (c) 2002-2016, Erik de Castro Lopo <erikd@mega-nerd.com>
+** Copyright (c) 2002-2021, Erik de Castro Lopo <erikd@mega-nerd.com>
 ** All rights reserved.
 **
 ** This code is released under 2-clause BSD license. Please see the
-** file at : https://github.com/erikd/libsamplerate/blob/master/COPYING
+** file at : https://github.com/libsndfile/libsamplerate/blob/master/COPYING
 */
 
+
+
+
+#include <assert.h>
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
+#include <math.h>
 
-#include "config.h"
-#include "float_cast.h"
 #include "common.h"
+#include "config.h"
```

Comment:
Rather than adding this include manually, I'd suggest leaving it how it originally was:
https://github.com/libsndfile/libsamplerate/blob/a147b6dc4046deda22b2aa772b348717de772e2b/src/src_sinc.c#L9-L11
And adding the `HAVE_CONFIG_H` flag to the preprocessor defs in the vs project file.  As all other files have a similar check

Reply:
I can put them back in, but I wonder why we need them? They certainly weren't in there for the previous version and it won't work if the file isn't there so it seems silly to make it optional

Reply:
Right, `HAVE_CONFIG_H` seems to be a new thing in the latest version, and I'll be honest -- I have no idea what the use-case is without the config.  But modifying the original code as little as possible will make future dependency bumps easier (won't have to remember the edits we did), up to you.

Reply:
Fair enough :) Well I'll leave it for now, hopefully we won't need to upgrade again :P